catalogue/webapp/test/utils/dates.test.ts

```diff
@@ -583,3 +592,73 @@ describe('isLibraryOpen: determines whether the library is open on a given day',
     }
   });
 });
+
+describe('works correctly with a non-London timezone', () => {
```

Comment:
These two tests both fail if you don't have the patch in the second commit applied.